radixdlt/src/integration/java/com/radixdlt/consensus/simulation/synchronous/RandomLatencyTest.java

```diff
@@ -28,54 +32,49 @@
  * may be found.
  */
 public class RandomLatencyTest {
+	// use a maxLatency of 20x the min latency since we know a round can take up to
+	// atleast 6x transmission time. 20x so that we can hit these cases more often
+	private final int minLatency = 10;
+	private final int maxLatency = 200;
+	// the minimum latency per round is determined using the network latency
+	// a round can consist of 6 * max_transmission_time (MTT)
+	private final int trips = 6;
+	private final int synchronousTimeout = maxLatency * trips;
+
+	private Builder bftTestBuilder = BFTSimulatedTest.builder()
+		.randomLatency(minLatency, maxLatency)
+		.setGetVerticesRPCEnabled(false)
+		.pacemakerTimeout(synchronousTimeout) // Since no syncing needed 6*MTT required
+		.checkLiveness("liveness", synchronousTimeout, TimeUnit.MILLISECONDS)
+		.checkSafety("safety")
```

Comment:
Seeing these strings repeated very often makes me think that a default overload in BFTSimulatedTest builder for these common checks would be good

Reply:
I think that just hides what we want to test.

Reply:
Maybe I wasn't being clear. My proposal is not to move any of the checks to the builder, but just to have there be a default for the check *name*, like this:
```suggestion
		.checkSafety()
```

Reply:
I agree with this.  There's no meaningful use case for adding the safety check twice right now (as well as others), so would be more consistent and less error prone if it was added with a fixed name/key.

Reply:
Have to disagree here. It's less error prone if you can see upfront what test names you are using, especially if you are going to use those same names to check for specific results. It's not about use cases at this level but more about readability (like builder patterns).